pkgs/misc/cups/drivers/fxlinuxprint/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, lib, fetchzip, dpkg, autoPatchelfHook, cups }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  name = "fxlinuxprint-${version}";
+  version = "1.1.2-1";
+
+  src = fetchzip {
+    url = "https://onlinesupport.fujixerox.com/driver_downloads/fxlinuxpdf112119031.zip";
+    sha256 = "1mv07ch6ysk9bknfmjqsgxb803sj6vfin29s9knaqv17jvgyh0n3";
+    curlOpts = "--user-agent Mozilla/5.0";  # HTTP 410 otherwise
+  };
+
+  nativeBuildInputs = [ dpkg autoPatchelfHook ];
+  buildInputs = [ cups ];
+
+  buildPhase = ":";  # nothing to build
+
+  installPhase = ''
+    mkdir -p $out
+    dpkg -x $src/fxlinuxprint_${version}_${debPlatform}.deb $out
+
+    mv $out/usr/* $out
```

```suggestion
    mv usr/* $out
```

now the archive shouldn't be unpacked into $out

---

done in 58453ef967ed9d6f6a32101d4177fd0675f2523c